pkg/provider/kubernetes/crd/traefik/v1alpha1/ingressroutetcp.go

```diff
@@ -53,11 +54,12 @@ type TLSStoreTCPRef struct {
 
 // ServiceTCP defines an upstream to proxy traffic.
 type ServiceTCP struct {
-	Name             string `json:"name"`
-	Namespace        string `json:"namespace"`
-	Port             int32  `json:"port"`
-	Weight           *int   `json:"weight,omitempty"`
-	TerminationDelay *int   `json:"terminationDelay,omitempty"`
+	Name             string                 `json:"name"`
+	Namespace        string                 `json:"namespace"`
+	Port             int32                  `json:"port"`
+	Weight           *int                   `json:"weight,omitempty"`
+	TerminationDelay *int                   `json:"terminationDelay,omitempty"`
+	ProxyProtocol    *dynamic.ProxyProtocol `json:"proxyProtocol,omitempty"`
```

Comment:
is the dependecy to dynamic ok here?

Reply:
Yes it's fine here. We are already having such dependencies in `pkg/provider/kubernetes/crd/traefik/v1alpha1/ingressroute.go` for the `dynamic.ResponseForwarding`.

Reply:
yes